demo/server/src/modules/keycloak/index.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="./keycloak.d.ts" />
+
+import * as express from 'express';
+import * as session from 'express-session';
+import Keycloak = require('keycloak-connect');
+
+// Express Session Configuration.
+const sessionOpts = {
+      secret: process.env.FH_COOKIE_SECRET || 'raincatcher',
```

Comment:
process.env.FH_COOKIE_SECRET is RHMAP specific. Let's use profile based config for this.